firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -1046,3 +1056,194 @@ uint8_t TCMDEXEC_reboot(const char *args_str, TCMD_TelecommandChannel_enum_t tcm
     return 0;
 }
 
+/// @brief Send artibrary configuration commands to a peripheral (MPI, GPS, CAMERA)
+/// @param args_str 
+/// - Arg 0: UART port to send data to
```

Comment:
Example? Number? Name? Both/either (like the EPS channels)

Reply:
`Arg 0: UART port to send data to: MPI, LORA, GPS, CAMERA, EPS (case insensitive)`

Updated to support peripheral name in a case insensitive format. Sample function call now looks like:
`CTS1+uart_send_bytes_hex(MPI,00110011...)!`